modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/services/liferay/accounts/useGetAccountByERC.ts

```diff
@@ -34,6 +26,6 @@ export default function useGetAccountByERC(
 					Liferay.authToken,
 			  ]
 			: null,
-		(url, token) => liferayFetcher<Account>(url, token)
+		(url, token) => liferayFetcher<AccountEntry>(url, token)
```

Comment:
did you tested a yarn build? since you changed a type, maybe this can broke in somewhere.

Reply:
yep, no problems